.github/workflows/_helm.yml

```diff
@@ -0,0 +1,71 @@
+name: Package helm charts
+
+on:
+  workflow_call:
+
+env:
+  HELM_VERSION_TO_INSTALL: 3.14.3
+
+jobs:
+  package-helm-charts:
+    name: Package and Push Helm Chart
+    runs-on: ubuntu-latest
+    environment: release
```

Comment:
Could we remove this so it doesn't spam PRs with deployment messages? Maybe we could keep it if the publish was in its own job, but I am not sure it is adding much.